display/graph/gauge/gauge.go

```diff
@@ -0,0 +1,196 @@
+	"github.com/mum4k/termdash/linestyle"
+	"github.com/mum4k/termdash/terminal/termbox"
+	"github.com/mum4k/termdash/terminal/terminalapi"
+	"github.com/mum4k/termdash/widgets/gauge"
+	"github.com/mum4k/termdash/widgets/text"
+)
+
+const RootID = "root"
+
+type metricColumn struct {
+	title  *text.Text
+	gauges []*gauge.Gauge
+}
+
+func newMetricColumn(column []*schema.SelectedRecord, config *dashboard.MetricConfig) (*metricColumn, error) {
+	var ret metricColumn
+	var maxValue int
+
+	t, err := text.New()
+	if err != nil {
+		return nil, err
+	}
+	if err := t.Write(config.Title, text.WriteCellOpts(cell.FgColor(cell.ColorRed))); err != nil {
+		return nil, err
+	}
+	ret.title = t
+
+	if config.Condition.Order == schema.OrderDes {
+		temp, err := strconv.Atoi(*(column[0].Value))
+		if err != nil {
+			return nil, err
+		}
+		maxValue = temp
+	} else if config.Condition.Order == schema.OrderAsc {
+		temp, err := strconv.Atoi(*(column[len(column)-1].Value))
+		if err != nil {
+			return nil, err
+		}
+		maxValue = temp
+	}
+
+	for _, item := range column {
+		strValue := *(item.Value)
+		v, err := strconv.Atoi(strValue)
+		if err != nil {
+			return nil, err
+		}
+
+		if config.AggregationNum != "" {
+			aggregationNum, convErr := strconv.Atoi(config.AggregationNum)
+			if convErr != nil {
+				return nil, convErr
+			}
+			strValue = fmt.Sprintf("%.4f", float64(v)/float64(aggregationNum))
+		}
+
+		g, err := gauge.New(
+			gauge.Height(1),
+			gauge.Border(linestyle.Light),
+			gauge.Color(cell.ColorMagenta),
+			gauge.BorderTitle("["+strValue+"]"),
+			gauge.HideTextProgress(),
+			gauge.TextLabel(item.Name),
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		if err := g.Absolute(v, maxValue); err != nil {
+			return nil, err
+		}
+		ret.gauges = append(ret.gauges, g)
+	}
+
+	return &ret, nil
+}
+
+func layout(columns ...*metricColumn) ([]container.Option, error) {
+	const GaugeNum = 6
+	const TitleHeight = 10
+
+	gaugeHeight := math.Floor((100 - TitleHeight) / GaugeNum)
+	var metricColumns []grid.Element
+
+	for _, c := range columns {
+		var column []grid.Element
+		column = append(column, grid.RowHeightPerc(TitleHeight, grid.Widget(c.title)))
+
+		for i := 0; i < GaugeNum; i++ {
+			column = append(column, grid.RowHeightPerc(int(gaugeHeight), grid.Widget(c.gauges[i])))
+		}
+		metricColumns = append(metricColumns, grid.ColWidthPerc(25, column...))
+	}
+
+	builder := grid.New()
+	builder.Add(
+		grid.RowHeightPerc(10),
+		grid.RowHeightPerc(70, metricColumns...),
+	)
+
+	gridOpts, err := builder.Build()
+	if err != nil {
+		return nil, err
+	}
+	return gridOpts, nil
+}
+
+func Display(metrics [][]*schema.SelectedRecord) error {
+	t, err := termbox.New()
+	if err != nil {
+		return err
+	}
+	defer t.Close()
+
+	c, err := container.New(
+		t,
+		container.ID(RootID),
+	)
+	if err != nil {
+		return err
+	}
+
+	var columns []*metricColumn
+
+	configs, err := dashboard.LoadConfig("assets/config/Dashboard.Global.json")
```

Comment:
According to me previous comment, `"assets/config/Dashboard.Global.json"` should be given by the users in the command options/flags